### ISSUE TITLE
Paroxysmal nocturnal dyspnea

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9861]
**Preferred term label:**
Paroxysmal nocturnal dyspnea

**Synonyms**
Nighttime awakening due to dyspnea; shortness of breath waking the patient from sleep

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Heart failure

**Your nano-attribution (ORCID)**
https://www.ncbi.nlm.nih.gov/medgen/C1956415

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034807
